index.html

```diff
@@ -4,7 +4,293 @@
     <meta charset="UTF-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Aleo&family=Josefin+Sans:wght@300;400;700&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="css/style.css" />
     <title>Travel With Us</title>
   </head>
-  <body></body>
+  <body>
+    <main>
+      <section class="section1">
+        <h1 class="section1__title section1_title">Bhromaon</h1>
+        <p class="section1__subtitle section1_subtitle">Safe Travel Anywhere</p>
```

Comment:
тут треба передивитись назви класів
Якщо у тебе напр. є блок `section1` і ти хочеш задати стилі його елемента `title`, то можна обійтись класом `section1__title`. А якщо ти знаєш, шо такий `title `буде в декількох місцях в інших блоках, то вже треба буде додати клас .title і прописати стилі туди, а в `section1__title `лишити тільки маржини, ширину і т.п.
```suggestion
        <h1 class="section1__title">Bhromaon</h1>
        <p class="section1__subtitle">Safe Travel Anywhere</p>
```